spec/support/fake_analytics.rb

```diff
@@ -95,7 +95,11 @@ def browser_attributes
   attributes ||= {}
 
   match do |actual|
-    expect(actual).to be_kind_of(FakeAnalytics)
+    if actual.instance_of? FakeAnalytics
+      expect(actual).to be_kind_of(FakeAnalytics)
+    else
+      expect(actual).to be_kind_of(IrsAttemptsApiTrackingHelper::FakeAttemptsTracker)
+    end
```

Comment:
This doesn't feel quite right, but I'm missing some context. I don't think FakeAttemptsTracker stuff should be ending up in FakeAnalytics.

98-99 feels potentially unsafe; this changes the existing expectations to only expect FakeAnalytics if it's FakeAnalytics.

Could we just change our usage of `have_logged_event` to a `have_received(:track_event)` and avoid this?

Reply:
Done!